src/components/selection_input/selection_input.ts

```diff
@@ -179,12 +189,19 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
   }
 
   getColor(range: SelectionRange) {
+    if (!this.isEnabled(range)) {
+      return "#fff";
+    }
     if (!range.color) {
       return "";
     }
     return cssPropertiesToCss({ color: range.color });
   }
 
+  isEnabled(range: SelectionRange): boolean {
+    return range.enabled ?? true;
+  }
```

Comment:
If you change the boolean from `isEnabled` to `isDisabled`, you could remove this function (undefined is falsy).